CMakeLists.txt

```diff
@@ -1025,6 +1025,16 @@ if(BUILD_TESTING)
       add_custom_target(fips_specific_tests_if_any)
     endif()
 
+    # Add macho parser tests if FIPS and on MacOS
+    if(FIPS AND APPLE)
+      add_custom_target(
+        macho_parser_tests
```

Comment:
Do you actually need a separate target for this? The target is not configured to depend on the what it executes, so in principle it's misconfigured. You can fix that.

Or you can just define a `test_macho_parser` as a dependency to e.g.  `fips_specific_tests_if_any` in `/util/fipstools/inject_hash/tests/CMakeList.txt`. Maybe?

Then you would not need this target and `ninja run_tests` would execute the macho tests.